core/src/main/java/org/pentaho/cdf/storage/StorageEngine.java

```diff
@@ -56,14 +57,22 @@ public static Operation get( String operation ) {
     }
   }
 
-  public static synchronized StorageEngine getInstance() {
+  public static synchronized StorageEngineInterface getInstance() {
```

Comment:
As my comment on the main case and the main PR, I don't believe this should be a public method. It's only here to facilitate testing, it should be package-protected not public.